portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that stores comments. */
+@WebServlet("/comments")
+public class CommentsServlet extends HttpServlet {
+
+  private List<String> comments = new ArrayList<>();
+
+  /* Returns comments as a JSON string. */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException {
+    // Convert comments to JSON using Gson
+    String json = new Gson().toJson(comments);
+
+    // Send json as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /* Stores given comment in the comments ArrayList */
```

Comment:
Add period to end of comment to be consistent with other function-level comments. I recommend all comments end with a period, but that is not required as long as it's consistent across comment types.

Reply:
Ok, I've fixed up the comments in both java files and also had them follow javadoc style guideline.